spec/features/site_admin_management_spec.rb

```diff
@@ -11,6 +13,28 @@
     expect(page).to have_css('td', text: user.email)
   end
 
+  describe 'copy email addresses' do
+    it 'displays the email addresses of site admins' do
+      expect(page).to have_css('div#all_users', text: user.email)
+    end
```

Comment:
Please combine these assertions into a single it block.  This test is very slow because it does several http requests for each `it` block.  If you need more fine grained tests, consider using a view spec.